keys.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Execute the aws sts assume-role command and store the output in a variable
+output=$(aws sts assume-role --role-arn arn:aws:iam::546240550610:role/Admins --role-session-name tf --serial-number <your_mfa_id> --token-code <your_token_code> --profile <your_profile>)

```

Comment:
hmm i think we need to be able to pass profile name to script and get --serial-number --role-arn from profile information. Something like
`aws configure  --profile dos14 get role_arn`

Reply:
also we need to as user for token code